warehouse/static/js/warehouse/utils/webauthn.js

```diff
@@ -185,23 +187,28 @@ export const ProvisionWebAuthn = () => {
     const resp = await fetch(
       "/manage/account/webauthn-provision/options", {
         cache: "no-cache",
+        credentials: "same-origin",
       }
     );
 
     const credentialOptions = await resp.json();
     const transformedOptions = transformCredentialOptions(credentialOptions);
-    const credential = await navigator.credentials.create({
+    await navigator.credentials.create({
       publicKey: transformedOptions,
-    });
-    const transformedCredential = transformCredential(credential);
+    }).then(async (credential) => {
+      const transformedCredential = transformCredential(credential);
 
-    const status = await postCredential(label, transformedCredential, csrfToken);
-    if (status.fail) {
-      populateWebAuthnErrorList(status.fail.errors);
-      return;
-    }
+      const status = await postCredential(label, transformedCredential, csrfToken);
+      if (status.fail) {
+        populateWebAuthnErrorList(status.fail.errors);
+        return;
+      }
 
-    window.location.replace("/manage/account");
+      window.location.replace("/manage/account");
+    }).catch((error) => {
```

Comment:
why not use try/catch style instead?

Reply:
I was under the impression that await-then-catch was the idiomatic way to do this kind of pattern with async code in ES6. But N.B. that I'm not much of a JS programmer.